### DISCUSSION TITLE
exclusive selection.

### DISCUSSION THREAD

**Discussion by Curthaldor** [Serene-Arc/bulk-downloader-for-reddit#438] [Q&A] [answered]
Many search terms appear to be inclusive by default. Is there a way to make overlapping terms exclusive instead? For example, right now "--upvoted and --multireddit mymulti" would download all posts that fit either of the two search terms. Is there a way to isolate only the posts that qualify for both?

**Comment by Serene-Arc** [comment, marked as answer]
As far as I am aware, this is not possible through the Reddit API. A good rule of thumb is that if you can't do it through the website, you can't do it through the API either. The BDFR mainly works by collating various lists of submissions from sources from Reddit itself, there's no current ability to provide any sort of logic to the parsing of these lists.
